File: USACOContest/2020Jan/wormsort/wormsort.cpp
/*
ID: ulysses4
LANG: C++
PROG: wormsort
*/
#include <cstdio>
#include <cstring>
char PROG[] = "wormsort";

void actualMain() {
	// write programs here
}

int main() {
	int size = sizeof(PROG);
	char in[size + 3], out[size + 4];
	freopen(strcat(strcpy(in, PROG), ".in"), "r", stdin);
	freopen(strcat(strcpy(out, PROG), ".out"), "w", stdout);
	actualMain();
	fclose(stdin);
	fclose(stdout);
	return 0;
}

File: USACOContest/2020Jan/wormsort/wormsort.java
/*
ID: ulysses4
LANG: JAVA
PROG: wormsort
*/
import java.io.*;
import java.util.*;
public class wormsort {
	private static final String PROG = "wormsort";
	
	private static void actualMain() {
	
	}
	
	private static InputStream in;
	private static PrintStream out;
	private static Scanner scanner;
	public static void main(String[] args) throws IOException {
		in = new FileInputStream(PROG + ".in");
		out = new PrintStream(PROG + ".out");
		scanner = new Scanner(in);
		actualMain();
		in.close();
		out.close();
		scanner.close();
	}
}
